### DISCUSSION TITLE
How to get more then 1000 rows of any one table

### DISCUSSION THREAD

**Discussion by kais509** [supabase/supabase#1742] [Questions] [answered]
How do I get more than 1000 rows of a table?

**Comment by soedirgo** [comment, marked as answer]
Hey @kais509, you can adjust this from the dashboard in Settings > API > Settings > Max Rows.

**Reply by ijaspreetsinghh** [reply]
i have set max rows limit to 5001 but still getting only 1000 rows. 
[is it something like free vs paid plan limit] ?
![image](https://user-images.githubusercontent.com/44422128/147866584-2d85ece1-6a2d-4f98-ab03-6888fd96400c.png)

**Reply by soedirgo** [reply]
No it shouldn't be, this looks like a regression. Refer to the issue Gary linked.

**Reply by qirong77** [reply]
change the dashboard settings and also restart the project works for me

**Comment by GaryAustin1** [comment]
There is this bug report...
https://github.com/supabase/supabase/issues/4544
